src/ds/d500/d500-md.h

```diff
@@ -0,0 +1,508 @@
+﻿// License: Apache 2.0. See LICENSE file in root directory.
```

Comment:
Can we change this file to d585s-md.h?
D555 doesn't have this md for examle

Reply:
ok